pom.xml

```diff
@@ -445,6 +445,7 @@
                     <excludes>
                         <exclude>**/*IT.java</exclude>
                         <exclude>**/*IntegrationTest.java</exclude>
+                        <exclude>**/*ManualTest.java</exclude>
```

Comment:
I don't see any class ending with `*ManualTest` - is this needed?

Reply:
I added it because I wrote a manual test to check this issue - it's not necessary right now, but it can be useful in future if someone would like to write it's own `ManualTest`. In the other hand I can remove it if you think it's unnecessary